sdk/storage/azure-storage-blob/tests/test_page_blob.py

```diff
@@ -4,10 +4,11 @@
 # license information.
 # --------------------------------------------------------------------------
 import os
+import requests
```

Comment:
Sorry I didn't catch this in the first review, but we don't really need tests for Page Blob. One test in Block Blob should be enough. I think we can probably just delete the new tests here. (don't forget to delete the recordings)